.travis.yml

```diff
@@ -2,26 +2,15 @@ language: python
 
 python:
 - 3.5
+- 3.8
 
 envs:
-- TOXENV=django111
-- TOXENV=django20
-- TOXENV=django21
 - TOXENV=django22
 - TOXENV=quality
 
 matrix:
-  include:
-    - python: 2.7
-      env: TOXENV=quality
-    - python: 2.7
-      env: TOXENV=django111
-    - python: 3.6
-      env: TOXENV=django22
-    - python: 3.7
-      env: TOXENV=django22
+  allow_failures:
```

Comment:
Is there a particular reason to allow failures of python 3.8? it seems to me that it is not needed anymore

Reply:
Removed